### DISCUSSION TITLE
MDK1 will soon be retired and replaced completely with MDK2

### DISCUSSION THREAD

**Discussion by malware-dev** [malware-dev/MDK-SE#324] [Announcements]
I am currently working on an installer tool to make it easier to install MDK2. Once this is done, MDK1 will be fully retired. 

I have not yet decided exactly what I want to do; whether I'll replace the MDK code in this repo or transition to the other currently hidden mdk2 repo, but most likely I will push the MDK1 code to a legacy branch, and replace all the code with MDK2 and retain this site. It is probably the simpler solution.

We are on a timeframe of weeks rather than months before this happens.

**Comment by mkaito** [comment]
If the codebase is significantly different with no historical dependency on the previous version, I would archive this repo and publish the new one.

**Reply by malware-dev** [reply]
Ordinarily I would. But... the wiki... and the fact that people know this address

**Reply by malware-dev** [reply]
For the record though.
Advice and opinions is kinda the reason why I mentioned that in particular 😄

**Comment by THDigi** [comment]
My suggestion as I mentioned in PB channel too:
* This repo gets archived + link to MDK2
* MDK2 repo for reports and whatnot
* This repo's wiki:
  * Pages that got moved to SE would link to that (maybe even collapsible old content to be more obvious that it's been moved but without erasing it, [like so](https://github.com/THDigi/SE-ModScript-Examples/wiki/Whitelist-and-checking-it-outside-of-the-game))
  * Pages that are still used (like the huge generated docs ones) could be kept here, maybe you'll move those to SE wiki eventually too, but no point in finding them a new home.
  * And pages like VS setup would be moved to SE wiki too I imagine (when that installer is done and such)
